### PR TITLE
[DDW-844] Fix Storybook knobs

### PR DESCRIPTION
**Pull request by marcin-mazurek**
This PR fixes Storybook knobs which are breaking when using the `timemachine` library. 

## Testing Checklist

- [Slack QA thread](https://input-output-rnd.slack.com/messages/GGKFXSKC6)
- [ ] Test

---

### Basics
- [ ] PR assigned to the PR author(s) 
- [ ] `input-output-hk/daedalus-dev` and `input-output-hk/daedalus-qa` assigned as PR reviewers
- [ ] If there are UI changes, Alexander Rukin assigned as an additional reviewer
- [ ] PR has appropriate labels (`release-vNext`, `feature`/`bug`/`chore`, `WIP`)
- [ ] PR link is added to a Jira ticket, ticket moved to In Review
- [ ] PR is updated to the most recent version of the target branch (and there are no conflicts)
- [ ] PR has a good description that summarizes all changes
- [ ] PR contains screenshots (in case of UI changes)
- [ ] CHANGELOG entry has been added to the top of the appropriate section (*Features*, *Fixes*, *Chores*) and is linked to the correct PR on GitHub
- [ ] There are no missing translations (running `yarn manage:translations` produces no changes)
- [ ] Text changes are proofread and approved (Jane Wild / Amy Reeve)
- [ ] Japanese text changes are proofread and approved (Junko Oda)
- [ ] Storybook works and no stories are broken (`yarn storybook`)
- [ ] In case of dependency changes `yarn.lock` file is updated

### Code Quality
- [ ] Important parts of the code are properly commented and documented
- [ ] Code is properly typed with flow
- [ ] React components are split-up enough to avoid unnecessary re-renderings
- [ ] Any code that only works in main process is neatly separated from components

### Testing
- [ ] New feature/change is covered by acceptance tests
- [ ] New feature/change is manually tested and approved by QA team
- [ ] All existing acceptance tests are still up-to-date
- [ ] New feature/change is covered by Daedalus Testing scenario
- [ ] All existing Daedalus Testing scenarios are still up-to-date

### After Review
- [ ] Update Slack QA thread by marking it with a green checkmark

### CODE REVIEW

**Review by mchappell — APPROVED** [review]

**Review by mchappell — APPROVED** [review]
Thanks Marcin, good eye for detail here 👍

File: storybook/addons.ts [hunk 1]
```diff
@@ -1,3 +1,4 @@
+// date override must be imported and set up before any modules that rely on the Date object
```
Comment (szymonmaslowski) [inline storybook/addons.ts:1]: Is this comment still needed here?

**Review by szymonmaslowski — APPROVED** [review]

**Review by lucas-barros — APPROVED** [review]

File: storybook/addons.ts (hunk 1 above)
Comment (marcin-mazurek) [reply storybook/addons.ts:1]: Good catch. No longer needed.

**Comment by DominikGuzei** [comment]
@marcin-mazurek how does this fix the storybook knobs? I just tried the [netlify storybook build](https://deploy-preview-2824--daedalus-storybook.netlify.app/) and the knobs are still not updating in real-time … or did I misunderstand the scope of this PR?

**Comment by marcin-mazurek** [comment]
@DominikGuzei good point, @renanvalentin pointed that out earlier today: https://github.com/input-output-hk/daedalus/pull/2824#discussion_r796533851 I just wasn't aware we also create Netlify builds. I'm on it.

**Comment by marcin-mazurek** [comment]
@renanvalentin @DominikGuzei Storybook seems to be fixed now: https://deploy-preview-2824--daedalus-storybook.netlify.app/?path=/story/wallets-summary--wallet-summary

Also, Chromatic Storybook deployments have frozen date: https://61afa6b2acfbf7003ae248a3-aksceteuak.chromatic.com/?path=/story/wallets-transactions--wallet-tokens-transactions-list

image

Hope the issue is now resolved, but please have another look just in case.

**Comment by marcin-mazurek** [comment]
@renanvalentin I've added another way to trigger Chromatic builds - with a GitHub PR label. I wasn't able to dispatch a workflow from Actions tab, I was getting this error: https://github.com/input-output-hk/daedalus/runs/5028481582?check_suite_focus=true

Please have a look and let me know your thoughts?

**Comment by renanvalentin** [comment]
@marcin-mazurek Great job ! I think using label as a trigger is an elegant solution and we can see the chromatic action on the github actions summary. 🎉

**Review by renanvalentin — APPROVED** [review]

**Review by DominikGuzei — APPROVED** [review]
great work @marcin-mazurek 🎉

**Pull request merged**